### ISSUE TITLE
Register extension for Firefox & Chrome

### ISSUE THREAD

**Issue by cKlee** [skohub-io/skohub-extension#8]
I'm am not sure if I can not find the option for the development mode or if my institution hides this option. I was able to install the extension temporarily for debugging, not permanently.
![extension](https://user-images.githubusercontent.com/849235/75318931-e54d1300-586b-11ea-94ac-09af5ac11151.png)
Would be more easy to install a packed version.
![2020-02-26 07_49_00-Add-ons-Verwaltung](https://user-images.githubusercontent.com/849235/75319287-a7042380-586c-11ea-9651-4664fd873645.png)

**Comment by literarymachine** [comment]
There isn't really a way around this for [unsigned extensions](https://support.mozilla.org/en-US/kb/add-on-signing-in-firefox), it is similar for Chrome. We should indeed publish to the Firefox and Chrome marketplaces to make it easy to install the extension.

**Comment by literarymachine** [comment]
@acka47 could you publish the latest [release](https://github.com/hbz/skohub-extension/releases) to [Chrome Web Store](https://developer.chrome.com/webstore/publish) and [submit it as a Firefox add-on](https://extensionworkshop.com/documentation/publish/submitting-an-add-on/)? This should be done by someone from hbz, I believe.

**Comment by acka47** [comment]
I registered the extension for Firefox, it should appear at https://addons.mozilla.org/de/firefox/addon/skohub-extension/ at some point. Chrome extension to follow.

**Comment by acka47** [comment]
The Firefox add-on has now been approved, see https://addons.mozilla.org/en-US/firefox/addon/skohub-extension/. \o/

However, before being approved I got the following response that said we would have to do more for being approved:

> Please expand this add-on listing. Describe the purpose and features/interface changes/actions of this add-on in more detail. Also attach images demonstrating the features and showing the operation of the add-on. For suggestions on how to create a better listing see https://extensionworkshop.com/documentation/develop/create-an-appealing-listing/ .
Also explain who will want to use this add-on.

**Comment by acka47** [comment]
The Chrome plugin wasn't approved:

> Ihre Veröffentlichungsanfrage für Google Chrome-Artikel "SkoHub" mit der 
> ID ghalhmcgaicdcpmdicinaegnoanfmggd wurde abgelehnt, da er nicht unseren 
> Richtlinien entspricht.
> 
> Wenn Sie eine Version dieses Artikels bereits im Chrome Web Store 
> veröffentlicht haben, ist diese hiervon nicht betroffen und weiterhin 
> verfügbar.
> 
> 
> Dieser Artikel entspricht nicht dem folgenden Abschnitt unserer 
> Programmrichtlinien 
> <https://developer.chrome.com/webstore/program_policies>:
> 
> 
> Spam oder Platzierung im Store
> 
>   * Der Artikel hat ein leeres Beschreibungsfeld oder fehlende Symbole
>     oder Screenshots und scheint verdächtig zu sein.
> 
> 
> Sobald Ihr Produkt den Richtlinien des Chrome Web Store entspricht, 
> können Sie eine erneute Veröffentlichung Ihres Produkts im Chrome Web 
> Store-Entwickler-Dashboard 
> <https://chrome.google.com/webstore/developer/dashboard> beantragen. Ihr 
> Produkt wird auf Einhaltung der Richtlinien überprüft, bevor es wieder 
> veröffentlicht wird.
> 

I think we should just write the blog post about skohub-editor (https://github.com/hbz/skohub-editor/issues/17) and then will have enough material to incorporate or link to in the plugin descripition...

**Comment by acka47** [comment]
The Chrome webstore to register an extension really sucks. For one, they want screenshots in 1200 x 800. I took me some time to make it happen. What [others said](https://superuser.com/questions/1263519/how-to-create-a-1280x800-screenshot-of-a-chrome-browser-viewport) (using shutter etc.) didn't work out for me. In the end I found out that I have a resolution (1440:900) with the right aspect ration (16:10). I made a screenshot with this resolution and converted it to the required resolution with `$ convert screenshot.png -resize 1280x800 screenshot-resized.png`.
